_config.yml

```diff
@@ -26,7 +26,7 @@ logoalt: 18F logo
 # Navigation
 # List links that should appear in the site sidebar here
 navigation:
-- text: 18F Delivery: Partnership Playbook
+- text: 18F Delivery":" Partnership Playbook
```

Comment:
Ah, the entire string after `text:` needs to be wrapped, not just the `:` character by itself. Same for the one below, and the `navigation:` entry in `_config.yml` needs it, too.